unit_testing/test_labels.py

```diff
@@ -139,11 +139,30 @@ def test_compute_mean_squared_error():
     for x, y, z, _ in src.getNonZeroCoordinates():
         if z < 5:
             ref.data[x, y, z+2] = src.data[x, y, z]
-
     mse = sct_labels.compute_mean_squared_error(src, ref)
     assert mse == 1.1547005383792515
 
 
+def test_compute_mean_squared_error_warning(caplog):
+    src = fake_3dimage_sct()
+    ref = src.copy()
+
+    src.data = np.where(src.data == ref.data[0, 0, 0], 1500, ref.data)
+
+    sct_labels.compute_mean_squared_error(src, ref)
+    # Cannot use f-string in assert, I needed to create a variable before
+    string_form = f'Label mismatch: Labels [{ref.data[0,0,0]}] present in reference image but missing from input image.'
+    assert 'Label mismatch: Labels [1500.0] present in input image but missing from reference image.' in caplog.text
```

Comment:
I think this assertion might need to be updated to use `string_form`.

Reply:
Like use inp.data[0,0,0] ?

Reply:
Changed it, tell me if that is what you had in mind !

Reply:
Ah sorry, I more meant the variable `string_form` was created but not used. The two lines above are near-duplicates.  :sweat_smile:

Reply:
Oh, I am just blind. I missed that `string_form` was indeed being used at the time. My bad. :sweat_smile: 

Well, either way, it's all good now!